infra/recipes/azure-sandbox/config/kubernetes-services.tf

```diff
@@ -0,0 +1,17 @@
+resource "azurerm_kubernetes_cluster" "sandbox" {
+    name = "sandbox-k8s"
+    resource_group_name = azurerm_resource_group.sandbox.name
+    location = azurerm_resource_group.sandbox.location
+    dns_prefix = "sandboxk8s"
+
+    default_node_pool {
+      name = "default"
+      node_count = var.node_count
+      vm_size = "Standard_D2_v2"
```

Comment:
Is this configurable?

Reply:
What do you mean by this? I chose it somewhat arbitrarily based on some examples I found

Reply:
I meat to say we have other vm size as well that we can choose from. So this should be configurable thorough variable?

Reply:
I see. I will add it as a variable.